utils/constants.ts

```diff
@@ -68,7 +71,10 @@ export const TOKEN_SYMBOLS_MAP = {
       [CHAIN_IDs.POLYGON]: "0x8f3Cf7ad23Cd3CaDbD9735AFf958023239c6A063",
       [CHAIN_IDs.BOBA]: "0xf74195Bb8a5cf652411867c5C2C5b8C2a402be35",
       [CHAIN_IDs.ARBITRUM]: "0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1",
+      [CHAIN_IDs.BASE]: "0x50c5725949A6F0c72E6C4a641F24049A917DB0Cb",
       [CHAIN_IDs.GOERLI]: "0x5C221E77624690fff6dd741493D735a17716c26B",
+      // [CHAIN_IDs.GOERLI]: "0x11fE4B6AE13d2a6055C8D9cF65c55bac32B5d844", // Uncomment for Optimism/Base testnet usage.
```

Comment:
Yep, there are multiple DAI addresses on Goerli, depending on which L2 testnet you're trying to work with. To test base goerli, I had to add this one and comment the other one.

Instead of changing it, I just added the new one and commented it out in case anyone needs to use it in the future.